src/Fetch.js

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2016 The Emscripten Authors
+ * Copyright 2025 The Emscripten Authors
```

Comment:
We don't update copyright years in existing files.

Reply:
Sorry I misunderstood your comment, I will update them.